### ISSUE TITLE
Dropdown: Screen reader is not announcing dropdown option label when using keyboard Up/Down arrow to navigate over options.

### ISSUE THREAD

**Issue by jennylgh** [primefaces/primevue#2768]
### Describe the bug

When dropdown overlay is open, use keyboard Up or Down arrow to navigate over options, screen read should read out option labels. 
Currently screenreader does not read label.
It is reproducible on Dropdown [documentation page](https://primefaces.org/primevue/dropdown).

### PrimeVue version

3.13.1

### Vue version

3.x

### Language

ALL

### Build / Runtime

TypeScript

### Browser(s)

All browsers

### Steps to reproduce the behavior

1. Enable screen reader. If on Mac, use Command + F5 to enable VoiceOver. If on Windows, install and run NVDA.
2. Use keyboard Tab to move focus to Dropdown. Press Space bar to open dropdown.
3. Use keyboard Up and Down arrow to navigate over dropdown options, and listen to screen reader.

### Expected behavior

Screen reader should read out option label as user is using Up/Down arrow key to move over options.

**Comment by tugcekucukoglu** [comment]
Accessibility for Dropdown is under the development now. With next version, it will be fixed.

Thanks,
